### DISCUSSION TITLE
Where to find some good explanation for telegraf ?

### DISCUSSION THREAD

**Discussion by vatsal-fpl** [telegraf/telegraf#1633] [Q&A]
I have just started working with telegraf. I was able to configure it with dialogflow but I want to have a better understanding of telegraf methods, how it works. I have found everywhere but couldn't get any good documentation. If there's any tutorial,books,blogs please suggest some. Thank You

**Comment by kdabwl** [comment]
how about you type e.g. ?q=telegraph+JavaScript+tutorial into search engine ...

**Reply by vatsal-fpl** [reply]
I have looked for it but could not get any good explanation.

**Reply by kdabwl** [reply]
when I click github for no email then I mean no email, and not to violate MY choice.

what you feel is good for you is not topic in discussions, you are not even thankful for help.

**Comment by MKRhere** [comment]
While we're revamping the docs, check this out: https://github.com/telegraf/telegraf/discussions/1586
(Accidentally responded with work account earlier)

**Reply by vatsal-fpl** [reply]
Thank you for this answer @MKRhere
